themes/tdr/login/login-otp.ftl

```diff
@@ -0,0 +1,23 @@
+<#import "template.ftl" as layout>
+<@layout.registrationLayout; section>
+    <#if section="header">
+        ${msg("doLogIn")}
+    <#elseif section="form">
+        <form id="otp-login-form" class="${properties.kcFormClass!}" action="${url.loginAction}"
```

Comment:
There is the the `class="${properties.kcFormClass!}"` on the `<form/>` element. Is this still outputting to a `kc-...` class value? If so probably needs changing aswell

Reply:
Sorry, missed that one. The form doesn't need a class so I've just taken it out.